lib/Utils/LLVMIRToSrc.cpp

```diff
@@ -243,21 +243,21 @@ std::string getModuleIDFromIR(const llvm::Value *V) {
   return "";
 }
 
-bool SourceCodeInfo::empty() const noexcept { return SourceCodeLine.empty(); }
+bool SourceCodeInfo::empty() const noexcept { return true; }
```

Comment:
This looks not good. If a SourceCodeInfo is considered empty, the ICFG export functionality will try to skip it entirely. Have you made sure that this does not happen?

At least on my system, the LLVMBasedICFGExportTest fails for the SourceCode tests